src/net.cpp

```diff
@@ -1996,15 +1996,16 @@ void CConnman::ThreadMessageHandler()
                 continue;
 
             // Receive messages
```

Comment:
Nit: move `Receive messages` comment into the if block (the block covers both receiving and sending messages. Having the comment outside the block suggests it's just for receiving messages)